### ISSUE TITLE
Макеты

### ISSUE THREAD

**Issue by artemkurylev** [IKaSeI/RPIS_PAGE#2]
Здесь оставляем комментарии по поводу макетов

**Comment by Polikarpova** [comment]
Я добавила ввод номера телефона в макет страницы регистрации

**Comment by artemkurylev** [comment]
Что делать с изменением пароля? Оставляем как есть или будем добавлять эту возможность?

**Comment by Polikarpova** [comment]
Мне кажется, что не стоит. Я считаю, что надо сконцентрироваться на основных функциях сервиса.

**Comment by artemkurylev** [comment]
Возможно стоит добавить ссылку на профиль заявителя в краткое описание, в остальном у меня особо идей нет

**Comment by Polikarpova** [comment]
Так. Я добавила кнопку просмотра на чужие заявки. Чтобы пользователь мог увидеть больше информации. Изначально, надо было нажимать на изображение, но идея дурацкая.

Могу добавить такую же кнопочку, рядом с кнопкой просмотра. Потому что не знаю, куда ещё ссылку туда впихнуть

**Comment by artemkurylev** [comment]
Почему дурацкая?

**Comment by Polikarpova** [comment]
Мне кажется это не такая уж и явная функция. Кто-то догадается, кто-то случайно тыкнет, а кто-то ничего не поймёт и психанёт ;D

**Comment by artemkurylev** [comment]
Тогда и в моих заявках надо это изменить это

**Comment by Polikarpova** [comment]
Да. Не успела ещё. У меня сайт поехал

**Comment by Polikarpova** [comment]
Так. Добавила кнопку просмотра и в "Мои заявки"
